Controller/CRUDController.php

```diff
@@ -1368,6 +1368,16 @@ protected function preList(Request $request)
     }
 
     /**
+     * @param Request $request
+     *
+     * @return array
+     */
+    protected function getObjectIDFromRequest(Request $request)
```

Comment:
Why do you think this should be `final`? If I make this `final` then I can't override it which would defeat the reason I made it in the first place.